packages/opentelemetry-context-asynchooks/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@opentelemetry/context-asynchooks",
+  "version": "0.0.1",
+  "description": "OpenTelemetry Asynchooks-based Context Propagation",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "repository": "open-telemetry/opentelemetry-js",
+  "scripts": {
+    "test": "nyc ts-mocha -p ./tsconfig.json test/**/*.ts",
+    "tdd": "yarn run test -- --watch-extensions ts --watch",
```

Comment:
This could be `yarn test -- --watch-extensions ts --watch`, leave out the `run` in this command.

Reply:
I've removed it but it's was the same in the core package, we might want to update it there too